code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -13,6 +13,17 @@
 	origin_tech = "combat=1;engineering=1"
 	attack_verb = list("robusted")
 	hitsound = 'sound/weapons/smash.ogg'
+	var/image/hinges = "single_hinge"

```

Comment:
This is not image and you never actually set it to image so don't type it as such.